docs/source/models/pic.rst

```diff
@@ -7,9 +7,81 @@ The Particle-in-Cell Algorithm
+   
+   \nabla \cdot \mathbf{B} &= 0
+   
+   \nabla \times \mathbf{E} &= -\frac{\partial \mathbf{B}} {\partial t}
+   
+   \nabla \times \mathbf{B} &= \mu_0\left(\sum_s \mathbf{J}_s + \varepsilon_0 \frac{\partial \mathbf{E}} {\partial t} \right)
+   
+for multiple particle species :math:`s`.
+:math:`\mathbf{E}(t)` represents the electic, :math:`\mathbf{B}(t)` the magnetic, :math:`\rho_s` the charge density and :math:`\mathbf{J}_s(t)` the current density field.
+
+Except for normalization of constants, PIConGPU implements the governing equations in SI units.
+
+Relativistic Plasma Physics
+---------------------------
+
+The 3D3V particle-in-cell method is used to describe many-body systems such as a plasmas.
+It approximates the Vlasov--Maxwell--Equation
+
+.. math::
+
+   \partial_t f_s(\mathbf{x},\mathbf{v},t) + \mathbf{v} \cdot \nabla_x f_s(\mathbf{x},\mathbf{v},t)
+  + \frac{q_s}{m_s} \left[ \mathbf{E}(\mathbf{x},t)  + \mathbf{v} \times \mathbf{B}(\mathbf{x},t) \right] \cdot \nabla_v f_s(\mathbf{x},\mathbf{v},t) =0\\
+
+with :math:`f_s` as the distribution function of a particle species :math:`s`, :math:`\mathbf{x},\mathbf{v},t` as position, velocity and time and :math:`\frac{q_s}{m_s}` the charge to mass-ratio of a species.
+The momentum is related to the velocity by :math:`\mathbf{p} = \gamma m_s \mathbf{v}`.
+
+The equations of motion are given by the Lorentz force as
+
+.. math::
+
+  \frac{\mathrm{d}}{\mathrm{d}t} \mathbf{V_s}(t) &= \frac{q_s}{m_s}  \left[ \mathbf{E}(\mathbf{X_s}(t),t) + \mathbf{V_s}(t) \times \mathbf{B}(\mathbf{X_s}(t),t) \right]\\
+ \frac{\mathrm{d}}{\mathrm{d}t} \mathbf{X_s}(t) &= \mathbf{V_s}(t) .
+
+.. attention::
+
+   TODO: write proper relativistic form
+
+where :math:`\mathbf{X_s}, \mathbf{V_s}` describe the ensemble of particle species' position and velocity.
+
+.. note::
+
+   Particles in a particle species can have different charge states in PIConGPU.
+   In the general case, :math:`\frac{q_s}{m_s}` is not required to be constant per particle species.
+
+Electro-Magnetic PIC Method
+---------------------------
+
+**Fields** such as :math:`\mathbf{E}(t), \mathbf{B}(t)` and :math:`\mathbf{J}(t)` are discretized on a regular mesh in Eulerian frame of reference (see [EulerLagrangeFrameOfReference]_).
+
+The distribution function :math:`f_s(\mathbf{x},\mathbf{v},t)` for **particles** is described in Lagrangian frame of reference.
+It is sampled with *markers*, sometimes referred to as *macro-particles*.
+These markers carry a spatial shape of order :math:`n` and a delta-distribution in momentum space.
+In most cases, these shapes are implemented as B-splines and are pre-integrated to *assignment functions* :math:`S` of the form:
+
+.. math::
+
+   S^0(x) = \big\{ \substack{1 \qquad \text{if}~0 \le x \lt 1\\ 0 \qquad \text{else}}
+
+   S^n(x) = \left(S^{n-1} * S^0\right)(x) = \int_{x-1}^x S^{n-1}(\xi) d\xi
+
+PIConGPU implements these up to order :math:`n=4`.
```

Comment:
Plus `The three dimensional marker shape is a multiplicative union of B-splines S(x,y,z) = S(x)S(y)S(z)`?